docs/src/main/paradox/routing-dsl/directives/parameter-directives/parameter.md

```diff
@@ -1,13 +1,5 @@
 # parameter
 
-@@@ div { .group-scala }
```

Comment:
I guess it wasn't super useful before and below we refer to the canonical `parameters`, so I guess that's good enough :+1: